mlir-tensorrt/executor/include/mlir-executor/Runtime/API/API.h

```diff
@@ -977,6 +991,7 @@ class RuntimeClient {
   PinnedMemoryAllocator pinnedMemoryAllocator;
   AllocTracker allocTracker;
   ResourceTracker resourceTracker;
+  std::set<DLManagedTensor *> dlPackTensors;
```

Comment:
Please use "llvm::DenseMap". Actually, if we do not need to do frequent lookups, then just tracking using a vector would be OK too. Depends on how often this map is going to be used. Check the notes in the LLVM programmer manual guide to guide the specific container choice.